components/inputs/test/input-date-range.visual-diff.js

```diff
@@ -253,13 +242,13 @@ describe('d2l-input-date-range', () => {
 					it('focus start', async function() {
 						await focusOnInput(page, '#min-max', startDateSelector);
 						const rect = await getRectTooltip(page, '#min-max');
-						await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+						await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect, captureBeyondViewport: false });
```

Comment:
These particular tests needed either `captureBeyondViewPort: false` or an extra `requestAnimationFrame`. Otherwise, the tooltip was only half displayed when the screenshot was taken.

Reply:
My best bet is that this is just because `captureBeyondViewport: false` has a few extra `await`s in it, so the browser has more time to finish painting:
https://github.com/puppeteer/puppeteer/blob/16b39de5ca5bdf70d8dbfe8168057991c6164bac/packages/puppeteer-core/src/common/Page.ts#L1400

If that's the case then throwing in an extra `requestAnimationFrame` may be more transparent. Maybe that could be added to the `focusOnInput` helper?